packages/react-native-media-player/android/src/main/java/com/yajananrao/mediaplayer/MediaPlayerService.java

```diff
@@ -0,0 +1,594 @@
+package com.yajananrao.mediaplayer;
+
+import android.app.NotificationChannel;
+import android.app.NotificationManager;
+import android.app.PendingIntent;
+import android.content.*;
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.media.AudioAttributes;
+import android.media.AudioFocusRequest;
+import android.media.AudioManager;
+import android.media.MediaPlayer;
+import android.net.ConnectivityManager;
+import android.net.NetworkInfo;
+import android.net.Uri;
+import android.os.Build;
+import android.os.Bundle;
+import android.os.Handler;
+import android.os.PowerManager;
+import android.support.v4.media.MediaBrowserCompat;
+import android.support.v4.media.MediaMetadataCompat;
+import android.support.v4.media.session.MediaSessionCompat;
+import android.support.v4.media.session.PlaybackStateCompat;
+import android.text.TextUtils;
+import android.util.Log;
+import android.widget.Toast;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.core.app.NotificationCompat;
+import androidx.core.app.NotificationManagerCompat;
+import androidx.media.MediaBrowserServiceCompat;
+import androidx.media.session.MediaButtonReceiver;
+
+import java.util.HashMap;
+import java.util.List;
+import java.lang.Thread;
+import java.lang.Runnable;
+
+public class MediaPlayerService extends MediaBrowserServiceCompat implements AudioManager.OnAudioFocusChangeListener {
+
+    private static final String TAG = "MediaPlayerService";
+    private static final int NOTIFICATION_ID = 121;
+    public static final String CHANNEL_ID = "com_yajananrao_mediaplayer";
+    private static final String CHANNEL_NAME = "Track Player";
+    private static final String ACTION_PROGRESS_UPDATE = "ACTION_PROGRESS_UPDATE";
+
+    private NotificationManager mNotificationManager;
+    private NotificationManagerCompat mNotificationManagerCompat;
+    private MediaPlayer mMediaPlayer;
+    private MediaSessionCompat mMediaSessionCompat;
+    private AudioFocusRequest focus;
+
+    final Object focusLock = new Object();
+
+
+    boolean resumeOnFocusGain = false;
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/yajananrao/Nadha/pullRequest?prid=5728824)